.gitignore

```diff
@@ -8,3 +8,5 @@ net.sf.eclipsecs.core/lib/metadata-gen-1.0-SNAPSHOT.jar
 .checkstyle
 target/
 show.patch
+patch-filters/
```

Comment:
Please rebase on latest master code, this line is not required.

Reply:
@romani Strange, it is rebased on latest master. https://github.com/gaurabdg/eclipse-cs/commits/meta-mainintegration

Reply:
Done